### DISCUSSION TITLE
How to get minimum/maximum allowed price for some skus?

### DISCUSSION THREAD

**Discussion by DearRude** [amzn/selling-partner-api-models#3758] [General] [answered]
I want to be able to get and change `minimum-seller-allowed-price` and `maximum-seller-allowed-price`. I already found out they could be changed/set by `feed` API.
Is there any way that I can get the current values of `minimum-seller-allowed-price` and `minimum-seller-allowed-price`?

**Comment by stefnats** [comment, marked as answer]
I think you should be able to do this with the [`getListingsItem` Call](https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference#getlistingsitem) from the Listings Items API
